codice/ddf
# [2.13.x] DDF-4300 UI fix for advanced queries when anyText input is blank

#### What does this PR do?

Applies UI fixes to Intrigue for handling anyText queries made with the different search forms to support legacy sources. Changes the default text input in basic and simple search forms to be blank. Changes the default `anyText` input in advanced forms to be `*`. For all three forms, the default query will be `anyText is like %`. Adding temporal/spatial criteria in basic forms will result in not defaulting to adding an `anyText is like %` filter in the query.

#### Who is reviewing it? 

@peterhuffer @pvargas @oconnormi @kcover @emmberk

#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.


@andrewkfiedler
@clockard

#### How should this be tested?

- Open Intrigue and start a search with an advanced form
  - Verify that there is a `*` value pre-filled in the `anyText` input
  - Click `Search` and verify that the resulting query is `anyText is like %`
  - Edit the search and clear the `anyText` input
  - Click `Search` and verify that the resulting query is `anyText is like ''`
- Start a search with a basic form
  - Verify that the text input box is blank
  - Click `Search` and verify that the resulting query is `anyText is like %`
  - Edit the search and verify that the text input box is still blank
- Start a search with a simple form
  - Verify that the text input box is blank
  - Click `Search` and verify that the resulting query is `anyText is like %`
  - Edit the search and verify that the text input box contains `*`

#### What are the relevant tickets?
[DDF-4300](https://codice.atlassian.net/browse/DDF-4300)

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
